### ISSUE TITLE
Python code availability

### ISSUE THREAD

**Issue by jluethi** [mpicbg-csbd/CSBDeep#3]
Hi there,

Am I missing something or is the python code currently not available? I've been reading the bioRxiv paper and you mentioned, linking to this repository:
> Code for network training and prediction (written in Python using Keras [48] and TensorFlow [49]) is publicly available.

Looking at the github repository, I can only find a readme, a config file & the docs.  
![screen shot 2018-01-09 at 18 03 58](https://user-images.githubusercontent.com/18033446/34733262-09ddedfc-f568-11e7-9511-466abf29b03c.png)

Will you be making the Python code become available as well?

Best,
Joel

PS: What I'm mostly curious to find out at the moment: What kind of loss function did you use to train the network? You reported the improvements in NRMSE & SSIM in figure 1. Was NRMSE or SSIM also in the loss function of the network?

**Comment by maweigert** [comment]
Hi, 

The python code for data generation and training will be released soon. Stay tuned! 

As we're generally predicting a (laplace) distribution at every pixel, we use its negative log-likelihood as loss, which is effectively a weighted MAE (see Supp. Notes p 47, eq 3.10).

**Comment by sziem** [comment]
Hi,
I would be very interested in the code as well!

PS: I really enjoyed reading the paper, but it also took me some time to find the loss function description in the supplementary material.

**Comment by hadim** [comment]
Hello. Any reason why the source code is not available yet?

**Comment by fjug** [comment]
Hello all, hello @hadim,

it is really funny you asked 11 hours ago, since we have been working hard on releasing the python code TODAY!
You should find it as of NOW on the csbdeep.bioimagecomputing.com website...

Please enjoy and we are very happy to receive feedback from you.

Twitter announcements will follow ASAP.

Best,
Florian
